dnscommon.r2py

```diff
@@ -519,9 +520,25 @@ def _read_single_answer(answer_index, dns_query_data):
 
   # There are a lot of different types of queries. We can parse all of these ones 
   # in the same way, though.
-  simple_answers = ['A', 'NS', 'CNAME', 'MD', 'MB', 'MF', 'MG', 'MR', 'MX', 'PTR']
+  simple_answers = ['A', 'CNAME', 'MD', 'MB', 'MF', 'MG', 'MR']
   if answer_type in simple_answers:
     read_index, resource_data['address'] = _parse_answer_address(read_index, dns_query_data)
+  elif answer_type == 'NS':
+    read_index, resource_data['nameserver'] = _parse_address(read_index,dns_query_data)
```

Comment:
Please combine all the answer types that are treated the same way under a single `elif` branch.